website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -3749,39 +3749,87 @@ the user ids from a `PCollection` of purchases one would write (using the `Selec
 purchases.apply(Select.fieldNames("userId"));
 {{< /highlight >}}
 
+{{< highlight python >}}
```

Comment:
I think this is the issue. Unfortunately our infra is considering "python" to be a separate language from "py"

```suggestion
{{< highlight py >}}
```

Reply:
ohh thanks